UI/forms/OBSAbout.ui

```diff
@@ -87,6 +87,22 @@
          </property>
         </widget>
        </item>
+       <item>
+        <spacer name="verticalSpacer_2">
+         <property name="orientation">
+          <enum>Qt::Orientation::Vertical</enum>
+         </property>
+         <property name="sizeType">
+          <enum>QSizePolicy::Policy::Fixed</enum>
```

Comment:
As far as I know, support for fully qualified enums was only backported to Qt 6.5/6.6. Ubuntu 24.04 still uses 6.4, so I'm not sure that it fully supports these, even though CI appears to be building. Ubuntu 24.10 and newer will use Qt 6.6+.